src/main/kotlin/org/opensearch/commons/alerting/action/DeleteCommentRequest.kt

```diff
@@ -0,0 +1,29 @@
+package org.opensearch.commons.alerting.action
+
+import org.opensearch.action.ActionRequest
+import org.opensearch.action.ActionRequestValidationException
+import org.opensearch.core.common.io.stream.StreamInput
+import org.opensearch.core.common.io.stream.StreamOutput
+import java.io.IOException
+
+class DeleteCommentRequest : ActionRequest {
+    val commentId: String
+
+    constructor(commentId: String) : super() {
+        this.commentId = commentId
```

Comment:
Nitpick: I noticed that the `DeleteCommentResponse` calls it's variable `id`. Not blocking, but you could probably rename this variable to `id` to keep them uniform, or stick with `commentId` if that's used elsewhere.